consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/model/breathingspace/BreathingSpace.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.moj.sdt.cmc.consumers.model.breathingspace;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Getter
+@ToString
+@NoArgsConstructor
+public class BreathingSpace {
```

Comment:
Should there be some way of setting the fields on this class (e.g. through a constructor)?  It's possible I'm missing something clever though!

Reply:
removed this class anyway